region/brinstar/green/Green Brinstar Main Shaft.json

```diff
@@ -946,6 +2481,37 @@
         }
       }
     },
+    {
+      "link": [7, 8],
+      "name": "Green Brinstar Main Shaft Moonfall Spark",
+      "notable": true,
+      "entranceCondition": {
+        "comeInShinecharged": {
+          "framesRequired": 150
+        }
+      },
+      "requires": [
+        "canMoonfall",
+        "canShinechargeMovementTricky",
+        {"shinespark": {"frames": 12}}
+      ],
+      "exitCondition": {
+        "leaveWithSpark": {}
+      },
+      "unlocksDoors": [
+        {"types": ["super"], "requires": []},
+        {"types": ["missiles", "powerbomb"], "requires": ["never"]}
+      ],
+      "note": [
+        "Come in with a shinecharge, and perform a moonfall off the ledge.",
+        "While falling, fire a shot at the door and immediately activate the shinespark wind-up, to horizontally spark through the door."
```

Comment:
You want to be moving forward when shooting.
And pressing up instead of jump worked a lot better for the shinespark wind-up.

I could land and jump with 170 frames, where this strat used 140 for me.  Activating it midair at the correct height was very hard.

Reply:
Interesting, you may have a faster way to get the moonfall off the ledge? Or were you maybe entering with non-zero momentum? 

I guess either way, it's still not quite enough frames to work with the vanilla door connection, so in Map Rando this would be an ignored notable.